flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/PartitionedFileWriteReadTest.java

```diff
@@ -755,7 +762,7 @@ private PartitionedFileWriter createPartitionedFileWriter(
 
     private PartitionedFileWriter createAndFinishPartitionedFileWriter() throws IOException {
         PartitionedFileWriter partitionedFileWriter = createPartitionedFileWriter(1, new int[0]);
-        partitionedFileWriter.finish();
+        partitionedFileWriter.finish().deleteQuietly();
```

Comment:
I think we should delete the partitioned file after the test finishes instead of at the beginning. How about make this change at the end of `testFinishPartitionedFileWriterTwice()` and `testWriteFinishedPartitionedFile()`?

Reply:
If it's not very straightforward to implement this, I would prefer to introduce a property PartitionedFile partitionedFile, and have each case assign a value to it. Then, delete it in the AfterEach method.
For example:
```
    private PartitionedFile partitionedFile;

    @AfterEach
    void after() {
        partitionedFile.deleteQuietly();
    }
```